jdaviz/configs/imviz/helper.py

```diff
@@ -114,9 +95,14 @@ def link_data(self, link_type='pixels', wcs_fallback_scheme='pixels', wcs_use_af
 
         .. warning::
 
-            Any markers added would be removed. You can add back the markers
-            manually using :meth:`add_markers`. During the markers removal,
-            pan/zoom will also reset.
+            Any markers added in default viewer would be removed automatically.
+            Any markers added in a manually created tiled viewer need to be
+            removed manually first using
```

Comment:
(minor wordsmithing)
```suggestion
            removed first using
```